google/cloud/credentials.py

```diff
@@ -102,8 +102,8 @@ def _get_signed_query_params(credentials, expiration, string_to_sign):
               signed payload.
     """
     if not hasattr(credentials, 'sign_blob'):
-        auth_uri = ('http://gcloud-python.readthedocs.io/en/latest/'
-                    'gcloud-auth.html#setting-up-a-service-account')
+        auth_uri = ('http://google-cloud-python.readthedocs.io/en/latest/'
```

Comment:
This seems to 404 for me.
http://google-cloud-python.readthedocs.io/en/latest/google-cloud-auth.html#setting-up-a-service-account